src/django/api/tests/test_v1_utils.py

```
@@ -33,6 +33,29 @@ def test_serialize_params_with_deep_object(self):
+        self.assertEqual(serialized_params['contributor_id'], [142, 44])
+        self.assertEqual(
+            serialized_params['contributor_type'], ['Brand / Retailer']
+        )
+        self.assertEqual(
+            serialized_params['sort_by'], 'number_of_contributors'
+        )
+
+    def test_serialize_params_with_invalid_contributor_id(self):
+        query_dict = QueryDict('', mutable=True)
+        query_dict.setlist('contributor_id', ['not-a-number'])
+        serialized_params, error_response = \
```

Ruff (0.15.21)

[warning] Unpacked variable `serialized_params` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)